### DISCUSSION TITLE
some questiones about the meaning of parameters in the software

### DISCUSSION THREAD

**Discussion by xixi-1995** [Open-Systems-Pharmacology/Forum#745] [Questions & Problems]
hi, all
I 'm confused about the meaning of  some parameters in the software, such as "percentile "about physiological parameter, 
 
![image](https://user-images.githubusercontent.com/54713645/67541308-1204e880-f71b-11e9-8c94-3326dc982b7b.png)
when I adjust the value, the percentile changes correspondingly, how to understand the meaning of percentile?

**Comment by msevestre** [comment]
@xixi-1995 
When you create an individual and select the species `Human`, you are also asked to select a `Population`
see [Creating Individual](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-individuals) 

More specifically

_For humans, the population parameters database that comes with PK-Sim® includes information on the dependence of anatomical and physiological parameters relevant for PK-Sim® such as organ weights, blood flow rates, or tissue composition on age, gender, body weight, body mass index, which have been collected in a comprehensive literature search. Thus, when creating an individual of a given Age, Weight, and Height, the parameters will be adjusted based on the information included in the underlying database. There are three classes of parameters that will be adjusted:_

 * Organ volumes
* Further parameters for which distribution data are available in the database, e.g. the hematocrit value.
 * The volume fractions, which are scaled using a global scale factor taken from the database


In your case, Gastric Emptying time is one of those parameters where distribution data is available. As such, the percentile shown here represents the percentile of the value in the population (based on your Age, Height and Weight selection)

When you change the value, the percentile updates automatically based on the underlying distribution. 

Hope that clarifies things,

**Comment by xixi-1995** [comment]
Dear  @msevestre ,thanks for your patiently explanation, actually I'm still confused.
Taking Gastric Emptying time as an example, when the value is set for 30min,the percentile is 50%, is that mean people with 30min Gastric Emptying time occupy for all 50% population I selection? but when I change the value to 50min,the percentile is 99%? from where I see, the percentile of each value should be 100% when sum them all up, is that right?
on the other hand, when I select different population with different age ,height or weight, the percentile of the same value seems to be the same, is there something wrong?
